docs/release_notes.md

```diff
@@ -1,4 +1,11 @@
 # Subscriptions API Release Notes
+<div class='post-date'>04 November 2020 - Invoice fields</div> 
+Merchants can add [invoice](invoice) for susbscription payments without user's information.
+Fields not mandatory: consumer_name, consumer_phone_number, consumer_address_lines.
```

Comment:
*"Fields that are now optional"

Reply:
By the way, isn't the term "property" more appropriate in this context instead of "field" ?

Reply:
From my understanding - merchants fill the fields and then properties get filled from the fields. Which naming should I choose?

Reply:
In the end I went with properties.

Reply:
@TadasAmbrazaitis merchants are sending a JSON with properties.